tests/client_app/CMakeLists.txt

```diff
@@ -1,7 +1,8 @@
 project(client_app)
 cmake_minimum_required(VERSION 3.0)
 
-find_package(libappimage REQUIRED)
-
 add_executable(client_app main.c)
 target_link_libraries(client_app libappimage)
+
+add_executable(client_app_static_linked main.c)
```

Comment:
The "test" here is to check if linking works? You should document that somewhere, e.g., above this line.